pages/reference-modules/4337/v0.3.0-1/operations/executeUserOp.mdx

```diff
@@ -0,0 +1,119 @@
+import { Tabs, Callout } from 'nextra/components'
+
+import LegacyCallout from '../../../../../components/callouts/LegacyCallout.mdx'
+import OnlySafeTxCallout from '../../../../../components/callouts/OnlySafeTxCallout.mdx'
+import ReentrancyCallout from '../../../../../components/callouts/ReentrancyCallout.mdx'
+import IrreversibilityCallout from '../../../../../components/callouts/IrreversibilityCallout.mdx'
+
+
+
+# `executeUserOp` (0x7bb37428)
+
+
+
+Executes a user operation provided by the entry point.
+
+
+
+
+Defined in [` Safe4337Module.sol`](https://github.com/safe-global/safe-modules/tree/4337/v0.3.0-1/modules/4337/contracts/Safe4337Module.sol#L172)
+
+
+## Usage
+
+{/* <!-- vale off --> */}
+
+<Tabs items={['example.sol']}>
+  <Tabs.Tab>
+    ```solidity
+    interface ISafe {
+        function executeUserOp(address to,uint256 value,bytes memory data,uint8 operation) external onlySupportedEntryPoint
```

Comment:
ubernit: If this is automatically generated, I would include some additional spaces:

```suggestion
        function executeUserOp(address to, uint256 value, bytes memory data, uint8 operation) external onlySupportedEntryPoint
```